test/fixtures/e2e/bento/lightbox.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script async src="https://cdn.ampproject.org/bento.js"></script>
+    <script
+      async
+      src="https://cdn.ampproject.org/v0/bento-lightbox-1.0.js"
+    ></script>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.ampproject.org/v0/bento-lightbox-1.0.css"
+    />
+  </head>
+  <body>
+    <button id="open-button">Open</button>
```

Comment:
nit: could you also add a close button & use the api to close it?

Reply:
sure - fyi there's already a close button that's in the lightbox because the lightbox covers "open" button

Reply:
updated